brilirs/src/interp.rs

```diff
@@ -172,10 +221,10 @@ enum Value {
   Float(f64),
   Char(char),
   Pointer(Pointer),
+  Atomic(usize),
```

Comment:
You might consider using the "NewType" idiom here to not mix this usize with the usize used for variable names.